plugins/eos/meson.build

```diff
@@ -25,5 +27,5 @@ shared_module(
   install : true,
   install_dir: plugin_dir,
   c_args : cargs,
-  dependencies : [ plugin_libs, flatpak ]
+  dependencies : [ plugin_libs, flatpak, ostree, ]
```

Comment:
not sure if the trailing comma here makes a difference

Reply:
Shouldn’t do. Meson ignores trailing commas (thank god).